docs/HowToFAQ/check-errors

```diff
@@ -3,6 +3,13 @@
 d=$(dirname ${BASH_SOURCE[0]})
 cd $d
 
+echo ">>>> REWRITER"
+
+csfiles="../../Source/DafnyCore/Rewriters/*.cs ../../Source/DafnyCore/Rewriters/*/*.cs"
+./check-errors-a Errors-Rewriter.md "$csfiles" ../../Source/DafnyCore/Rewriters/RewriterErrors.cs
+
+exit 0
```

Comment:
When do we run this?

Reply:
Well, it will be -- I'm using it to check all the error Ids, but it needs a bit of cleanup (and unification across all the files that use errorids) becfore I put it in CI.